WebApp-Connect-To-Azure-Sql-Database/Client/README.md

```diff
@@ -0,0 +1,780 @@
+---
+page_type: sample
+name: How to use an authenticated user's credentials for log-in to an Azure SQL Database from Blazor Web Server App
+services: active-directory
+platforms: dotnet
+urlFragment: ms-identity-dotnet-blazor-azure-sql
+description: This sample demonstrates how to use access token obtained from AAD for connecting to Azure SQL Server database as a user that is logged in into the application.
+languages:
+ - csharp
+ - t-sql
+products:
+ - aspnet-core
+ - blazor
+ - azure-active-directory
+ - azure-sql
+---
+
+# How to use an authenticated user's credentials for log-in to an Azure SQL Database from Blazor Web Server App
```

Comment:
Please keep the README outside the code folder as practiced with every other sample. It keeps the Github landing page work (they default to readme.md). Also this will not confuse those who expect MS samples to have a readme in the root and code in other sub-folders